content/docs/command-reference/exp/show.md

```diff
@@ -65,6 +64,42 @@ will be generated using the same data from the table.
 
 ![](/img/pcp_interaction.gif) _Parallel Coordinates Plot_
 
+### Paginating the output
+
+This command's output is automatically piped to
+[less](<https://en.wikipedia.org/wiki/Less_(Unix)>) if available in the terminal
+(the exact command used is `less --chop-long-lines --clear-screen`). If `less`
+is not available (e.g. on Windows), the output is simply printed out.
+
+> It's also possible to
+> [enable `less` on Windows](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
+
+> Note that this also applies to `dvc dag`.
+
+### Providing a custom pager
+
+It's possible to override the default pager via the `DVC_PAGER` environment
+variable. For example, the following command will replace the default pager with
+[`more`](<https://en.wikipedia.org/wiki/More_(command)>), for a single run:
+
+```dvc
+$ DVC_PAGER=more dvc exp show
+```
+
+For a persistent change, define `DVC_PAGER` in the shell configuration. For
+example in Bash, we could add the following line to `~/.bashrc`:
+
+```dvc
+$ export DVC_PAGER=more
+```
+
+If you'd like to print to stdout always without paging, you can set this
+variable to `cat`:
+
+```dvc
+$ export DVC_PAGER=cat
+```
```

Comment:
Let's not further expand on Linux methods? There are others but this doc should be about DVC IMO.

Also, this isn't in https://dvc.org/doc/command-reference/dag#providing-a-custom-pager ATM.

Reply:
Whole environment variable thing is about Linux. The method provides a way to skip the `--no-pager` option everytime. So I think it's a useful tip.

Reply:
Hey sorry for the delay...

We can improve the env var blocks later to include Windows Cmd (Windows also supports the same env vars I think).

This is a cmd ref, we want it to be more focused/shorter/self contained I think. Again, this note is a Linux tip.

Reply:
removed

Reply:
> We can improve the env var blocks later to include Windows Cmd

I'm making it clearer that these are Linux examples instead for now. See https://github.com/iterative/dvc.org/pull/3356